server/src/main/java/com/google/coffeehouse/servlets/LeaveClubServlet.java

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse.servlets;
+
+import com.google.coffeehouse.storagehandler.StorageHandlerApi;
+import com.google.coffeehouse.common.MembershipConstants;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * Servlet to leave a club from Http POST Request Body (in JSON format) containing a club ID
+ * and user ID that exists in the database, and return a success status 200 (OK).
+ */
+@WebServlet("/api/leave-club")
+public class LeaveClubServlet extends HttpServlet {
+
+  /** 
+   * The error string sent by the response object in doPost when the body of the 
+   * POST request cannot be used to leave a club for any reason.
+   */
+  public static final String BODY_ERROR = "- unable to parse body.";
+  /** The message to be logged when the body of the POST request cannot be parsed. */
+  public static final String LOG_BODY_ERROR_MESSAGE = 
+      "Body unable to be parsed in LeaveClubServlet: ";
+
+  /** 
+   * The error string sent by the response object in doPost when the body of the 
+   * POST request does not have a required field.
+   */
+  public static final String NO_FIELD_ERROR = "Missing \"%s\" field in JSON";
+  /** The message to be logged when the body of the GET request does not have required fields. */
+  public static final String LOG_INPUT_ERROR_MESSAGE =
+      "Error with JSON input in GetProfileServlet: ";
+  /** Name of the key in the input JSON that corresponds to the userId. */
+  public static final String USER_ID_FIELD_NAME = "userId";
+  /** Name of the key in the input JSON that corresponds to the clubId. */
+  public static final String CLUB_ID_FIELD_NAME = "clubId";
```

Comment:
Reference these from Person and Club (`Person.USER_ID_FIELD_NAME` and `Club.CLUB_ID_FIELD_NAME` are already defined).

Reply:
Got it! I'll make sure to change it in JoinClubServlet as well afterwards.